site/source/user-manual/backups/backup-setup/backup-synology.rst

```diff
@@ -1,8 +1,8 @@
 .. _cifs-synology:
 
-========================================
-Creating a LAN Shared Folder on Synology
-========================================
+==================
```

Comment:
This Synology page looks totally different than Linux and Windows. The latter have sections for "Create LAN Shared Folder" and "Connect Embassy". Each platform page should follow the same format.